perms/perms.go

```diff
@@ -212,6 +212,22 @@ var (
 			Entity: "universe",
 			Action: "read",
 		}},
+		"/rfqrpc.Rfq/UpsertAssetBuyOrder": {{
```

Comment:
Yeah, the "Upsert" definitely feels like the wrong word at this level...

Reply:
Changed to `Add...`